docs/otel/sdk-metrics.md

```diff
@@ -483,6 +484,8 @@ E.g. for Java the fully qualified classname SHOULD be used in this case.
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## Log metrics
```

Comment:
I just noticed that I forgot to add this heading in the previous PR, therefore sneaking it into this one.